src/test/java/org/owasp/esapi/reference/EncoderTest.java

```diff
@@ -914,11 +914,28 @@ public void testHtmlEncodeStrSurrogatePair()
     
     public void testHtmlDecodeHexEntititesSurrogatePair()
     {
-    	HTMLEntityCodec htmlCodec = new HTMLEntityCodec();
+        HTMLEntityCodec htmlCodec = new HTMLEntityCodec();
         String expected = new String (new int[]{0x2f804}, 0, 1);
         assertEquals( expected, htmlCodec.decode("&#194564;") );
         assertEquals( expected, htmlCodec.decode("&#x2f804;") );
     }
     
+    public void testUnicodeCanonicalize() {
```

Comment:
I think you should at least add a comment here to refer to the appropriate GitHub issue #.